examples/direct_fidelity_estimation.py

```diff
@@ -248,6 +263,11 @@ def parse_arguments(args):
                         type=int,
                         help='Number of trials to run.')
 
+    parser.add_argument('--n_clifford_trials',
+                        default=3,
+                        type=int,
+                        help='Number of trials for Clifford circuits')
```

Comment:
Can you provide a more descriptive help message for this? Mention tradeoffs between setting this too large / too small

Reply:
Added some comments.